package.json

```diff
@@ -84,20 +81,21 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "1.0.0",
-    "@edx/frontend-build": "^6.1.0",
+    "@edx/frontend-build": "^8.1.6",
```

Comment:
tinest of nits: pin the dependency since Renovate tries to keep everything pinned.